sources/shdr/UI.coffee

```diff
@@ -102,6 +109,13 @@ class UI
       .attr('data-index', key))
     no
 
+  addNewModel: (modelName, key) ->
+    list = $('#menu-models .menu-list')
+    button = $('<button>').addClass('menu-item')
+    list.append(button.clone().text(modelName)
+    .attr('data-index', key))
+    no
```

Comment:
urh I can't believe I used to use this pattern, my bad. Please use an explicit `return false` if you need it, otherwise omit the return statement. thanks!